web/blueprints/user/__init__.py

```diff
@@ -151,6 +151,11 @@ def user_show(user_id):
         'inverted': True,
         'saldo': balance,
     }
+    lock_unlock="lock" # lock_unlock is "lock" if user isn't locked --> show button to lock user | if user is locked, lock_unlock is the membership-ID
+    for membership in memberships_active.all():
+        if membership.group.name == "Gesperrt":
+            lock_unlock=membership.id
+            break
```

Comment:
Instead of hard-coding the string „Gesperrt“ (which the according group does not need to have!), you should reference the group that is given in the config as done [here](https://github.com/agdsn/pycroft/blob/de8ad9742904ef7c51ed9a8accb55c9ded40bd52/pycroft/lib/user.py#L428): the corresponding group would be `config.violation_group`.

Although this works, we can end the membership even simpler: The library function `remove_member_of` in `lib.user` only needs to know the group, not the specific membership or its id. Thus, it would be more fitting to:

- [ ] create a new viewpoint `/unblock` (see the `suspend` function for comparison)
- [ ] create a new library function `def unblock(user, processor)` (see the `suspend`) that calls `remove_member_of(user=user, group=config.violation_group, processor=processor)`
- [ ] Use this new endpoint in the template.